packages/edit-post/src/components/admin-notices/test/index.js

```diff
@@ -19,7 +19,9 @@ describe( 'AdminNotices', () => {
 					<p>My <strong>notice</strong> text</p>
 					<p>My second line of text</p>
 					<button type="button" class="notice-dismiss">
-						<span class="screen-reader-text">Dismiss this notice.</span>
+						<span class="components-visually-hidden">
```

Comment:
If we're keeping the original classname as per [this thead](https://github.com/WordPress/gutenberg/pull/20607#discussion_r388153911) then this needn't change either.